python/pip_install/pip_repository_requirements.bzl.tmpl

```diff
@@ -43,12 +43,40 @@ def _get_annotation(requirement):
     return _annotations.get(name)
 
 def install_deps(**whl_library_kwargs):
+    """Repository rule macro. Install dependencies from `pip_parse`.
+
+    Args:
+       **whl_library_kwargs: Additional arguments which will flow to underlying
+         `whl_library` calls. See pip_repository.bzl for details.
+    """
+
+    all_requirement_groups = %%ALL_REQUIREMENT_GROUPS%%
+
     whl_config = dict(_config)
     whl_config.update(whl_library_kwargs)
```

Comment:
nit: moving this to just before the for loop would make things a little easier to read. Right now we have a lot of things going on in the code here.